Sources/NIOHTTP2/ConcurrentStreamBuffer.swift

```diff
@@ -81,6 +81,14 @@ struct ConcurrentStreamBuffer {
         return nil
     }
 
+    func invalidateBuffer(reason: Error) {
+        for buffer in self.bufferedFrames {
```

Comment:
are we sure that whilst calling out we can't get any more (or changed) `self.bufferedFrame` ?

Reply:
I think yes (only called from handlerRemoved AFAICS)

Reply:
Yeah, I don’t think we can because of that call site. It’s a hard requirement of this code that that be true, and we’ll get crashes if I’m wrong, but I think I’m right.